Dart/src/com/jetbrains/lang/dart/ide/actions/DartEditorNotificationsProvider.java

```diff
@@ -105,11 +105,14 @@ public EditorNotificationPanel createNotificationPanel(@NotNull final VirtualFil
   private static class PubActionsPanel extends EditorNotificationPanel {
     private PubActionsPanel() {
       super(EditorColors.GUTTER_BACKGROUND);
-      myLinksPanel.add(new JLabel("Pub actions:"));
+
+      icon(DartIcons.Dart_16);
+      text("Pub actions");
+
```

Comment:
It does degrade a bit when the editor is very wide. I think I still prefer using an icon and `text()` - the icon is nice that have visually, and the text() call makes the notification have a similar look to other ones.

But I don't feel strongly about it, if you want to handle the wide monitor use case well. I agree that an icon wouldn't make sense if not using text().